samples/nativeSample/src/main/java/com/inmobi/nativead/sample/singlestrand/SingleStrandFragment.java

```diff
@@ -0,0 +1,191 @@
+package com.inmobi.nativead.sample.singlestrand;
+
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.util.DisplayMetrics;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.RatingBar;
+import android.widget.TextView;
+
+import com.inmobi.ads.InMobiAdRequestStatus;
+import com.inmobi.ads.InMobiNative;
+import com.inmobi.nativead.PlacementId;
+import com.inmobi.nativead.sample.R;
+import com.inmobi.nativead.utility.SwipeRefreshLayoutWrapper;
+import com.squareup.picasso.Picasso;
+
+/**
+ * Demonstrates the use of InMobiNativeStrand to place a single ad.
+ * <p/>
+ * Note: Swipe to refresh ads.
+ */
+public class SingleStrandFragment extends Fragment
```

Comment:
We no longer have "strands", so the class name and the class-level Javadoc needs to  change.